src/Common/tests/System/Runtime/Serialization/Formatters/BinaryFormatterHelpers.cs

```diff
@@ -71,5 +72,32 @@ public static void AssertRoundtrips<T>(T expected, params Func<T, object>[] addi
                 }
             }
         }
+
+        public static void AssertExceptionDeserializationFails<T>() where T : Exception
+        {
+            // .NET Core throws a PlatformNotSupportedException for deserializing many exceptions.
+            if (!RuntimeInformation.FrameworkDescription.StartsWith(".NET Core"))
```

Comment:
I was wondering couple times before why we haven't yet created on there. It will be definitely useful.